model/version.go

```diff
@@ -86,8 +86,24 @@ type Version struct {
 
 	// this is only used for aggregations, and is not stored in the DB
 	Builds []build.Build `bson:"build_variants,omitempty" json:"build_variants,omitempty"`
+
+	// StorageMethod describes how the parser project for this version is
+	// stored. If this is empty, the default storage method is StorageMethodDB.
+	StorageMethod ProjectConfigStorageMethod `bson:"storage_method" json:"storage_method,omitempty"`
```

Comment:
Can this be ProjectStorageMethod or something so the variable includes what we're discussing?